db/migrate/20220218144349_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :user_id
+      t.text :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+    add_index :posts, :user_id
```

Comment:
- [ ] To customize our foreign key we can do something like this -> ‘ add_reference :posts, :author, references: :user, foreign_key: true’. This will create an author_id column in the posts table as the foreign key. And we can update our User model with this code -> ‘  has_many :posts, foreign_key: 'author_id’. Kindly try this and let’s see how it goes. I do appreciate your comment but I think it is doable. Kindly visit this [link](https://guides.rubyonrails.org/active_record_migrations.html#changing-columns) if you want deeper understanding